examples/springboot/src/main/java/io/airbrake/weather/WeatherController.java

```diff
@@ -15,31 +13,31 @@
 public class WeatherController {
 
     @Autowired
-    WeatherService svc;
+    WeatherService weatherService;
 
     @GetMapping("/date")
     public String date() {
         
-        return svc.GetDate();
+        return weatherService.getDate();
     }
 
     @GetMapping("/locations")
     public String locations() {
        
-        return svc.GetLocations();
+        return weatherService.getLocations();
     }
 
     @GetMapping("/weather/{location}")
     public String weather(@PathVariable String location) {
     
-        return svc.GetWeather(location);
+        return weatherService.getWeather(location);
       
     }
 
     @GetMapping("/weather1/{location}")
-    public String weather1(@PathVariable String location) {
+    public String weatherTryCatch(@PathVariable String location) {
     
-        return svc.GetWeather1(location);
+        return weatherService.getWeatherTC(location);
```

Comment:
Please update this function name to getWeatherTryCatch

Reply:
Changed the function name. Now its working fine